services/minespace-web/src/tests/mocks/dataMocks.js

```diff
@@ -1957,3 +1957,26 @@ export const NOTICE_OF_DEPARTURE_DETAILS = {
     },
   ],
 };
+
+export const REGIONS = [
```

Comment:
There is a bit of a move todataMocks.tsx in common. Wondering if this should go there.

Reply:
Works for me!